R/char.R

```diff
@@ -198,5 +198,5 @@ vec_cast.character.pillar_char <- function(x, to, ...) {
 }
 #' @export
 vec_cast.pillar_char.character <- function(x, to, ...) {
-  vec_restore(x, to)
+  vec_restore(vec_proxy(x), to)
```

Comment:
hmm actually in this case this doesn't make much of a difference since cast methods are not inherited. So you're guaranteed to get a bare character. Sorry for misleading you here.